packages/fx-core/src/plugins/resource/appstudio/constants.ts

```diff
@@ -45,6 +45,8 @@ export const DEFAULT_COLOR_PNG_FILENAME = "color.png";
 export const DEFAULT_OUTLINE_PNG_FILENAME = "outline.png";
 export const MANIFEST_RESOURCES = "resources";
 export const APP_PACKAGE_FOLDER_FOR_MULTI_ENV = "templates/appPackage";
+export const RETRY_INTERVAL = 30000;
```

Comment:
Nit: better to comment it with unit, or name the const with unit, for example, `RETRY_INTERVAL_MS`.